### PR TITLE
chore: set CLR_END_DATE

### PR DESCRIPTION
**Pull request by thelostone-mc**
##### Description

- set CLR_END_DATE if not default to current.timezone 
- removed defaults and added null checks in place
- updated code to use CLR_END_DATE

### CODE REVIEW

File: app/grants/clr.py [hunk 1]
```diff
@@ -35,6 +35,12 @@
 PREV_CLR_START_DATE = dt.datetime(2020, 3, 23, 12, 0)
 PREV_CLR_END_DATE = dt.datetime(2020, 4, 7, 12, 0)
 CLR_START_DATE = dt.datetime(2020, 6, 15, 12, 0)
+CLR_END_DATE = dt.datetime(2020, 7, 3, 16, 0)
+
+try:
+    CLR_END_DATE
+except NameError:
```
Comment (owocki) [inline app/grants/clr.py:42]: what is this for

**Comment by owocki** [comment]
didnt test but LGTM

**Comment by owocki** [comment]
at high level

File: app/grants/clr.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/clr.py:42]: Just to be extra safe to ensure no one comments out `CLR_END_DATE` while round is active 
That check ensure if that variable is not defined -> set it to current timezone

**Pull request merged**